cns/hnsclient/hnsclient_linux.go

```diff
@@ -50,3 +50,9 @@ func DeleteHostNCApipaEndpoint(
 	networkContainerID string) error {
 	return nil
 }
+
+// DeleteHNSEndpointbyID deletes the HNS endpoint
```

Comment:
`hnsclient_linux` is nonsensical and should not exist, why is it like this 😢

Reply:
the file should not exist at all or we should rename it

Reply:
I was struggling for a half day to see why the new functions is undefined and then saw this "linux" file. We have to rename this!

Reply:
I think it's an old code!

Reply:
This file needs to go. Don't add anything to it, this is a bad pattern taht we shouldn't continue.

Reply:
I tried to delete this file but the Make will fail. Seems like it fails whenever we run the Make in Linux. It seems we can only import hcn package in platfrom specifi file (windows). Do we have an example in our code base for similar situation?
In CNI I know we are adding *Iml to functions and create platfrom specific file like _windows or _linux